backend/packages/Upgrade/src/api/controllers/ExperimentClientController.v5.ts

```diff
@@ -179,7 +179,7 @@ export class ExperimentClientController {
   ): Promise<Pick<ExperimentUser, 'id' | 'group' | 'workingGroup'>> {
     request.logger.info({ message: 'Starting the init call for user' });
     // getOriginalUserDoc call for alias
-    const experimentUserDoc = await this.experimentUserService.getUserDoc(experimentUser.id, request.logger);
+    const experimentUserDoc = await this.experimentUserService.getUserDoc(experimentUser.id, request.logger, 'init');
```

Comment:
Create a function in the controller which should return the experiment user doc if exist else should throw an error